lib/book.rb

```diff
@@ -0,0 +1,11 @@
+# The Book class represents a book with a title and an author.
+# It provides getters and setters for the title and author.
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
```

Comment:
* Kindly ensure `A method for adding rentals be present` which shows the relationship `* A book has many rentals` [Check requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md#:~:text=Create%20the%20has%2Dmany%20side%20of%20Book%20and%20Rental%20(a%20book%20has%20many%20rentals).) 
* Link to the example in a previous lesson: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship